std/getopt.d

```diff
@@ -459,6 +459,19 @@ GetoptResult getopt(T...)(ref string[] args, T opts)
     }
 }
 
+version(DIP1000)
```

Comment:
I'm not sure how we want to handle DIP1000 regressions, but we need a way to ensure things stay `@safe`

Reply:
By https://github.com/dlang/phobos/pull/6278 and thus gradually enabling -dip1000?

Reply:
But this `unittest` needs to be annotated with `@safe` when `-dip1000` is passed and `@system` when it's not to prevent any DIP1000 regressions. #6278 Doesn't have a mechanism for that.

Reply:
Argh fair enough - there still might be issues as long as `-deps` compiles all unittests of its dependencies :/
Anyhow, for the time being #6278 (or similar could set `DIP1000` by hand).